text/src/autogluon/text/automm/constants.py

```diff
@@ -42,6 +42,20 @@
 AVERAGE_PRECISION = "average_precision"
 LOG_LOSS = "log_loss"
 CROSS_ENTROPY = "cross_entropy"
+METRIC_MODE_MAP = {
+    ACC: MAX,
+    ACCURACY: MAX,
+    RMSE: MIN,
+    ROOT_MEAN_SQUARED_ERROR: MIN,
+    R2: MAX,
+    QUADRATIC_KAPPA: MAX,
+    ROC_AUC: MAX,
+    LOG_LOSS: MIN,
+    CROSS_ENTROPY: MIN,
+    PEARSONR: MAX,
+    SPEARMANR: MAX,
+}
+VALID_METRICS = METRIC_MODE_MAP.keys()
```

Comment:
Can we use `higher_is_better` instead of manually hard-coding? See to [torchmetrics](https://torchmetrics.readthedocs.io/en/stable/pages/overview.html#metrics-and-hyperparameter-optimization).

Reply:
I think this is out of scope in this PR, and the change could result in other changes in AutoMM. We can discuss it and improve it in a separate PR

Reply:
Yeah, we can refine this part in a separate PR. We can actually mark it as a TODO.

Reply:
Created an issue to track: https://github.com/awslabs/autogluon/issues/1799